<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>OperationsWidget</class>
 <widget class="QDockWidget" name="OperationsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>90</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string comment="Window title of tool box">Operations Toolbar</string>
  </property>
  <widget class="QWidget" name="toolGroup">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="newButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="openButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveAsButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/saveas.png</normaloff>:/icons/saveas.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="copyButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="cutButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="pasteButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="undoButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="redoButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="icon">
       <iconset resource="../data/app.qrc">
        <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../data/app.qrc"/>
 </resources>
</ui>

